### DISCUSSION TITLE
Vercel/Sass

### DISCUSSION THREAD

**Discussion by etsets2023** [vercel/community#3839] [Help]
### Page to Investigate

https://vercel.com/etsets2023/projeto-3-design-systems-mobile-first

### Steps to Reproduce

Vercel isn't carrying the stylizations after I started using Sass. HTML is linked with css file, all paws are on GitHub, do not know what to do. Can anyone help me?
Vercel sent me an email stating that "Deployment Failed"

![Captura de tela 2023-08-30 110655](https://github.com/vercel/community/assets/139820173/bea247ad-3133-4ec8-8c18-f28646e8308f)

**Comment by etsets2023** [comment]
On Github everything looks ok:

https://github.com/etsets2023/Projeto_3_Design_Systems_MobileFirst

**Comment by amyegan** [comment]
Do you see any more information in the [Build Logs](https://vercel.com/docs/deployments/logs#)? That should tell you more about why the build failed.

**Comment by etsets2023** [comment]
Hi, Can you help me?

[image: image.png]
